plugins/source/postgresql/client/list_tables.go

```diff
@@ -38,7 +38,12 @@ SELECT
 		ELSE false
 	END AS not_null,
     CASE
-		WHEN conkey IS NOT NULL AND (contype = 'p' OR contype = 'u') AND array_position(conkey, pg_attribute.attnum) > 0 THEN true
+		WHEN
+		    conkey IS NOT NULL
+		    AND contype = 'u'
```

Comment:
Don't we want to set it as true if it's a single PK?

Reply:
do we?

Reply:
Yes, since that implies the column is unique in Postgres

Reply:
9e4063ef7663bf23c021d8d67a5f3952e61106e2